### PR TITLE
Apply range_start optimization in SortedSet class

### PR DESCRIPTION
**Pull request by ddrmanxbxfr**
Same optimization as the queue class.

Compute the range size once.

### CODE REVIEW

File: lib/sidekiq/api.rb [hunk 1]
```diff
@@ -502,7 +502,7 @@ def each
 
       while true do
         range_start = page * page_size + offset_size
-        range_end   = page * page_size + offset_size + (page_size - 1)
+        range_end   = range_start + (page_size - 1)
```
Comment (krzysiek1507) [inline lib/sidekiq/api.rb]: @ddrmanxbxfr `( )` unnecessary ;)

Comment (ddrmanxbxfr) [inline lib/sidekiq/api.rb]: Uh uh, indeed. Fixing it in both classes

Comment (krzysiek1507) [inline lib/sidekiq/api.rb]: :+1:

**Comment by davydovanton** [comment]
@krzysiek1507 could you fix `( )` in #2771 too?

**Comment by ddrmanxbxfr** [comment]
I just did in this PR if you look at the changes.

Thanks!

**Comment by krzysiek1507** [comment]
@davydovanton you mean @ddrmanxbxfr ;)

**Comment by ddrmanxbxfr** [comment]
Both method we're patched in this PR. The one in #2771 and a new one in
sorted set class.

**Comment by davydovanton** [comment]
thanks!

**Pull request merged**